docs/best-practices/code-reviews/readme.md

```diff
@@ -1,129 +1,154 @@
-  ways:
+- **Sharing knowledge** helps development teams in several ways:
 
   - A CR explicitly communicates added/altered/removed functionality to
     team members who can subsequently build on the work done.
 
   - The committer may use a technique or algorithm that reviewers can
-    learn from.
+    learn from. More generally, code reviews help [raise the quality bar across the organization](https://blog.fullstory.com/what-we-learned-from-google-code-reviews-arent-just-for-catching-bugs/).
 
-  - Reviewers may possess knowledge about the code base that can help
+  - Reviewers may possess knowledge about programming techniques or the code base that can help
     improve or consolidate the change; for example, someone else may be
     concurrently working on a similar feature or fix.
 
+  - Positive Interaction and communication strengthens social bonds between
+    team members.
+
 - **Consistency** in a code base makes code easier to read and
   understand, helps prevent bugs, and facilitates collaboration
   between regular and migratory developer species.
 
 - **Legibility** of code fragments is hard to judge for the author
-  whose brainchild it is, and easy to judge for a reviewer who does
+  whose brain child it is, and easy to judge for a reviewer who does
   not have the full context. Legible code is more reusable, bug-free,
   and future-proof.
 
 - **Accidental errors** (e.g., typos) as well as **structural errors**
-  (e.g., dead code, logic or algorithm bugs) are often much easier to
-  spot for critical reviewers with an outside perspective. (Often,
-  it's a rather satisfying sport to find bugs...)
+  (e.g., dead code, logic or algorithm bugs, performance or architecture
+  concerns) are often much easier to
+  spot for critical reviewers with an outside perspective.
+  (Finding other people's bugs is often a rather satisfying sport ...)
+  Studies have found that even short and informal code reviews have significant
+  [impact on code quality and bug frequency](https://blog.codinghorror.com/code-reviews-just-do-it/).
+
+- **Security reviews** are required in some environments. They are performed
+  by application security engineers following industry-standard
+  procedures and frameworks.
 
-## What to Review
+
+## What to review
 
 There is no eternally true answer to this question and the right
-approach should be agreed on by each development team; some teams prefer
-to review *every* CR merged into the develop branch, while others will have a
+approach should be agreed on by each development team. Some teams prefer
+to review every change merged into the main branch, while others will have a
 "triviality" threshold under which a review is not required. The
 trade-off is between effective use of engineers' (both authors' and
-reviewers') time and maintaining code quality.
+reviewers') time and maintaining code quality. In certain regulatory environments,
+code review may be required even for trivial changes.
```

Comment:
We might consider a “when to review” comment as well — pre-merge, post-merge/pre-release, feature branch reviews but no aggregate review

Reply:
added, and moved the "stacked CR" content here too